include/caf/caf_overview.rst

```diff
@@ -0,0 +1,182 @@
+.. _caf_overview:
+
+Common Application Framework overview
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Common Application Framework (CAF) is a set of ready-for-use modules and events built on top of :ref:`event_manager`.
+Using CAF allows you to have a consistent event-based architecture in your application.
+
+One of the applications that are using the CAF modules in |NCS| is :ref:`nrf_desktop`.
+
+In an event-based application, parts of the application functionality are separated into isolated modules that communicate with each other using Event Manager's events.
+These events are defined either by CAF or by the application.
+They can be submitted by modules and other modules can subscribe and react to them.
```

Comment:
Not "can" but "are". Choosing CAF determines application structure to modular event based. User is forced to define modules when working with CAF as a backbone.